extensions/blocks/eventbrite/edit.js

```diff
@@ -274,34 +266,35 @@ class EventbriteEdit extends Component {
 	 * @returns {object} The UI displayed when user edits this block.
 	 */
 	render() {
-		const { attributes, addModalButtonStyles, removeModalButtonStyles, isSelected } = this.props;
-		const { url, style } = attributes;
+		const { attributes } = this.props;
+		const { eventId, url, style, __isBlockPreview } = attributes;
 		const { editingUrl, isResolvingUrl } = this.state;
 
 		let component;
 
 		if ( isResolvingUrl ) {
-			removeModalButtonStyles();
 			component = this.renderLoading();
 		} else if ( editingUrl || ! url || this.cannotEmbed() ) {
-			removeModalButtonStyles();
 			component = this.renderEditEmbed();
 		} else {
-			// Don't add / remove button styles if blocks aren't selected
-			// For example in previews
-			if ( isSelected ) {
-				if ( style === 'modal' ) {
-					addModalButtonStyles();
-				} else {
-					removeModalButtonStyles();
-				}
-			}
-
 			component = (
 				<>
 					{ this.renderBlockControls() }
-					{ style === 'modal' ? (
-						<ModalButtonPreview { ...this.props } />
+					{ style === 'modal' && ! __isBlockPreview ? (
```

Comment:
I think that as [`getBlockByExample` recursively calls itself](https://github.com/WordPress/gutenberg/blob/972a8d5848905ac6e1084acecd3d7a3dab1b888a/packages/blocks/src/api/factory.js#L544) with the `example` attributes for the inner blocks, that the problem here is that `jetpack/button` doesn't have an `example` in its settings. Not sure why you wouldn't be getting an error and it's just a hunch, but it might be worth trying.